templates/partials/banner.html

```diff
@@ -1,15 +1,14 @@
-<section id="hero" class="page-section first-section text-white">
+<section id="hero" class="banner-section first-section text-white">
     <div id="banner-image" class="d-block d-lg-none bg opacity-30 bg-position-center-center bg-attachment-fixed"></div>
```

Comment:
Banner image is no longer visible?

Reply:
It's a side-effect from my work on the nodes page.  It's not showing on there and both pages use the same template.

Reply:
Banner image is fixed.